src/test/java/hw3/Ex2Test.java

```diff
@@ -0,0 +1,58 @@
+package hw3;
+
+import Pages.DiffElementsPage;
+import Pages.HeaderPage;
+import Pages.MainPage;
+import org.assertj.core.api.JUnitSoftAssertions;
+import org.junit.Rule;
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.annotations.Test;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class Ex2Test extends AbstractClass{
+    @Rule
+    public JUnitSoftAssertions softAssertions = new JUnitSoftAssertions();
+
+    @Test
+    public void ex2Tests() {
+        MainPage mainPage = PageFactory.initElements(driver, MainPage.class);
+        HeaderPage headerPage = PageFactory.initElements(driver, HeaderPage.class);
+        DiffElementsPage diffElementsPage = PageFactory.initElements(driver, DiffElementsPage.class);
+        //step 2 - Assert Browser title
+        Assertions.assertEquals(driver.getTitle(), "Home Page");
+
+        //step 3 - Perform login
+        //step 4 - Assert Username is loggined
+        mainPage.login("Roman", "Jdi1234");
+        String userName = mainPage.getUserName().getText();
+        softAssertions.assertThat(userName).isEqualTo("ROMAN IOVLEV");
+
+        //step 5 - Open through the header menu Service -> Different Elements Page
+        headerPage.clickService();
+        headerPage.clickDiffEl();
+
+        //step 6 - Select checkboxes Water, Wind
+        diffElementsPage.clickWater();
```

Comment:
Here and similar places should be method with String parameter to click on element by its name.